### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by BelkoEvgeniy1987**
Homework-08

### CODE REVIEW

File: homework/belko-evgeniy/lesson-08/css/style.css [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+.img1 {
+    background-image: url(../images/img1.svg);
+}
+.img2 {
+    background-image: url(../images/img2.svg);
+}
+.img3 {
+    background-image: url(../images/img3.svg);
+}
+
```
Comment (AlexOrd) [inline homework/belko-evgeniy/lesson-08/css/style.css:70]: Empty lines.

**Review by AlexOrd — APPROVED** [review]

**Comment by AlexOrd** [comment]
Why 'Add files via upload '?

File: homework/belko-evgeniy/lesson-08/css/style.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    font-family: 'Raleway', sans-serif;
+    font-size: 17px;
+    font-weight: 300;
+    line-height: 25px;
+    box-sizing: border-box;
+    margin-top: 30px;
+    margin-bottom: 50px;
+    opacity: 0.8;
+}
+
```
Comment (Towko) [inline homework/belko-evgeniy/lesson-08/css/style.css:32]: too much spaces

File: homework/belko-evgeniy/lesson-08/css/style.css [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+    height: 60px;
+    border-radius: 10%;
+    background-color: #262626;
+    box-sizing: border-box;
+    position: relative;
+    left: 30px;
+    top: 30px;
+    text-align: center;
+    line-height:15px;
+    background-color: rgb(38,38,38,0.5);
```
Comment (Towko) [inline homework/belko-evgeniy/lesson-08/css/style.css:86]: background-color: rgb(38,38,38,0.5); - так работает?
background-color: rgba(38,38,38,0.5); - вот так правильно

**Review by Towko — APPROVED** [review]

**Review by alexnightowl — APPROVED** [review]

**Pull request merged**